Framework/Core/src/WorkflowHelpers.cxx

```diff
@@ -184,6 +184,7 @@ void WorkflowHelpers::injectServiceDevices(WorkflowSpec& workflow)
       switch (input.lifetime) {
         case Lifetime::Timer: {
           auto concrete = DataSpecUtils::asConcreteDataMatcher(input);
+          processor.options.push_back(ConfigParamSpec{ "period-" + input.binding, VariantType::Int, 1000, { "period of the timer" } });
```

Comment:
@ktf 
How do I specify in the code the period of timer after this fix? Putting the same option inside DataProcessorSpec with desired period results in throwing 'option is ambiguous and matches different versions of 'period-timer-publish'

Reply:
I probably should avoid pushing the option if it's already there... :-/